EVSL_1.0/TESTS/GEN/KPM_MMPLanR.c

```diff
@@ -129,10 +128,6 @@ int main() {
         fprintf(flog, "read_coo error for B = %d\n", ierr);
         exit(6);
       }
-      /*------------------ diagonal scaling for Acoo and Bcoo */
-      sqrtdiag = (double *)calloc(n, sizeof(double));
-      extractDiag(&Bcoo, sqrtdiag);
-      diagScaling(&Acoo, &Bcoo, sqrtdiag);
```

Comment:
That depends on whether we use Choleksy factor or LS polynomial in DOS? In general, the Cholesky version is much slower but more accurate.